editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -6069,10 +6064,21 @@ Node *CanvasItemEditorViewport::_make_texture_node_type(String texture_node_type
 }
 
 void CanvasItemEditorViewport::_notification(int p_what) {
+	if (p_what == NOTIFICATION_ENTER_TREE || p_what == NOTIFICATION_THEME_CHANGED) {
```

Comment:
Would be good to unify this with the switch below.

Reply:
E.g. by putting this in the switch, and using an if check for the connect/disconnect stuff.